lochness/config/__init__.py

```diff
@@ -4,20 +4,33 @@
 import yaml.reader
 import getpass as gp
 import cryptease as crypt
+import string
 
 logger = logging.getLogger(__name__)
 
+
 def load(f, archive_base=None):
     '''load configuration file and keyring'''
     logger.debug('loading configuration')
+
     with open(os.path.expanduser(f), 'rb') as fp:
         Lochness = _read_config_file(fp)
+
     if archive_base:
         Lochness['phoenix_root'] = archive_base
     if 'phoenix_root' not in Lochness:
         raise ConfigError('need either --archive-base or \'phoenix_root\' in config file')
     Lochness['phoenix_root'] = os.path.expanduser(Lochness['phoenix_root'])
     Lochness['keyring_file'] = os.path.expanduser(Lochness['keyring_file'])
+
+    # box file pattern strings from the config to string template
+    # regardless of the selected study in the args
+    if 'box' in Lochness.keys():
+        for box_study_name, box_study_dict in Lochness['box'].items():
```

Comment:
This could possibly eliminate the `_return_box_patterns_str_template` function

for _,study_dict in Lochness['box'].items():
  for _,modality_values in study_dict['file patterns'].items():
    for modality_dict in modality_values:
      modality_dict['pattern'] = string.Template(modality_dict['pattern'])

Reply:
Thanks, Tim for the comment! Took out `_return_box_patterns_str_template`, replacing your suggestion https://github.com/PREDICT-DPACC/lochness/pull/2/commits/13d98de6a64cf69f049b371c5c0d0ac44d85421c

& tested https://github.com/PREDICT-DPACC/lochness/pull/2/commits/79351b916b316ead90d575c1129e74705cb19910